packages/design-system/src/components/button/button.tsx

```diff
@@ -1,29 +1,29 @@
-import type { ButtonHTMLAttributes } from 'react';
-import { buttonStyle } from './button.css';
+import { cn } from '../../utils';
+import { buttonVariants } from './button.css';
+import SvgIcLink from '../../icons/src/IcLink';
 
-export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
-  variant?: 'primary' | 'secondary' | 'tertiary' | 'outline';
-  size?: 'xLarge' | 'large' | 'medium';
-  isDisabled?: boolean;
+interface ButtonProps {
```

Comment:
```suggestion
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
```
`<button>` 요소의 기본 속성을 모두 상속받아 사용할 수 있도록 타입을 지정해주면, 타입 안정성 및 유연성에서 더욱 좋은 코드가 될 것 같아요 !